xbmc/music/tags/ReplayGain.cpp

```diff
@@ -74,6 +75,35 @@ void ReplayGain::SetPeak(Type aType, float aPeak)
     m_data[index(aType)].SetPeak(aPeak);
 }
 
+std::string ReplayGain::Get() const
+{
+  if (!Get(ALBUM).Valid() && !Get(TRACK).Valid())
+    return StringUtils::Empty;
```

Comment:
use `return std::string()` the Empty is some legacy thing that should be removed